server/web/models.py

```diff
@@ -0,0 +1,30 @@
+from . import db
+from werkzeug.security import generate_password_hash, check_password_hash
+
+
+class Account(db.Model):
+    __tablename__ = 'account'
+    id = db.Column(db.Integer, primary_key=True)
+    companyName = db.Column(db.String(64), unique=True)
+    createDate = db.Column(db.DateTime)
+    active = db.Column(db.Boolean)
+
+    def __repr__(self):
+        return '<Account %r>' % self.companyName
+
+class User(db.Model):
```

Comment:
- Needs constructor
- make sure on constructor that `self.password = self.set_password(password)` to ensure plain text password is properly hashed
- snake case column names